nginx/templates/multi-channel.conf.tmpl

```diff
@@ -126,6 +126,26 @@ server {
     auth_basic_user_file    /etc/nginx/.htpasswd;
   {{- end }}
 
+    # let ICM handle everything static ICM related
+    location ~* ^/INTERSHOP/static.*$ {
+        allow all;
+        auth_basic off;
+
+        proxy_set_header Host $http_host;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+
+        proxy_no_cache true;
+        proxy_cache_bypass true;
+        add_header X-Cache-Status $upstream_cache_status;
+
+        proxy_pass {{ getenv "ICM_BASE_URL" }};
```

Comment:
With that the `ICM_BASE_URL` becomes mandatory as an environment variable for nginx, which it shouldn't.

Reply:
It now has a condition around this block. Does this look better?

By the way why does the next block use `$UPSTREAM_PWA` while it seams to do pretty much the same?
```
    # let ICM handle everything ICM related
    location ~* ^/INTERSHOP.*$ {
    ...
        proxy_pass {{ $UPSTREAM_PWA }};
    ...
```

Reply:
Yes, with the condition it works.

The next block is used for the `PROXY_ICM` feature in the PWA SSR container, that also activates when `SSR_HYBRID` is active. So the question is, why do you actually need the additional block in the setup to go directly to the ICM instead of just letting the PWA SSR delegate it.

What was the original error? It couldn't resolve static files? CORS errors?

Reply:
![statc_INTERSHOP_resources](https://user-images.githubusercontent.com/50741106/165099521-a119bd63-ee8d-428d-a485-4d084812327f.png)

I would say without this additional block it couldn't resolve static files and not CORS issues.
And it is probably right to directly let ICM handle the static files and leave out the SSR container also for performance reasons.

Reply:
👍